src/qubic.cpp

```diff
@@ -2902,14 +2902,30 @@ static void endEpoch()
 
         // Compute revenue of computors and arbitrator
         long long arbitratorRevenue = ISSUANCE_RATE;
+        constexpr long long issuancePerComputor = ISSUANCE_RATE / NUMBER_OF_COMPUTORS;
+        constexpr long long scalingThreshold = 0xFFFFFFFFFFFFFFFFULL / issuancePerComputor;
+        // maxRevenueScore for 300k ticks = ((7099 * 300000) / 676) * 300000 * 675
+        constexpr unsigned scalingFactor = 51200; // >= (maxRevenueScore300kTicks / 0xFFFFFFFFFFFFFFFFULL) * issuancePerComputor =(approx)= 51159.9

```

Comment:
need a `static_assert(MAX_NUMBER_OF_TICKS_PER_EPOCH <= 300000, "Redefine scalingFactor");` here so it can remind us when `MAX_NUMBER_OF_TICKS_PER_EPOCH` exceeds 300k before 128bit implementation.